employees/tests/test_views.py

```diff
@@ -254,3 +341,193 @@ def test_delete_report_view_should_delete_report_on_post(self):
         response = self.client.post(path=self.url)
         self.assertEqual(response.status_code, 302)
         self.assertFalse(Report.objects.filter(pk=self.report.pk).exists())
+
+
+class ProjectReportListTests(TestCase):
+    def _assert_response_contain_report(self, response, reports):
```

Comment:
could you please move this helper method at the very bottom of the class?

Reply:
:+1: done